Assets/Scripts/Puzzle/ExitGate.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class ExitGate : Gate {
+	[Space]
```

Comment:
Space?

Reply:
When writing this I was expecting a bit more to this script and wanted to separate it from the normal gate. Could probably be removed.